### PR TITLE
WIP - install Bulkrax and debugging

### PR DESCRIPTION
**Pull request by labradford**
references #3

### CODE REVIEW

File: env/.env.prod.db [hunk 1]
```diff
@@ -0,0 +1 @@
+POSTGRES_PASSWORD=hydraDock3r!
```
Comment (bkiahstroud) [inline env/.env.prod.db:1]: The filename implies that this is meant for a production environment. Given that: 

1. Why has this been added? 
2. If this is the prod value, is this now a leaked secret?

File: env/.env.prod.hydra [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+FEDORA_DEV_URL=http://fcrepo:8080/rest/
+FEDORA_PASSWORD=fedoraAdmin
+SOLR_DEV_URL=http://solr:8983/solr/hydra_dev/
+TERM=xterm
+REDIS_HOST=redis
+REDIS_PORT=6379 
+REDIS_URL_SIDEKIQ=redis://redis:6379/12
+MEMCACHED_HOST=memcached
+RAILS_ENV=development
+RACK_ENV=development
```
Comment (bkiahstroud) [inline env/.env.prod.hydra:13]: The filename implies that this is meant for a production environment. Given that: 

1. Why have these been added? 
2. If these are prod values, are these now leaked secrets?

File: env/.env.prod.db (hunk 1 above)
Comment (labradford) [reply env/.env.prod.db:1]: 1. You will need these files to seed the db
2. These were already in the repo in the env.dev.db so they aren't true production values
we can remove these before committing this MR, but they are here to get the app running.

File: env/.env.prod.hydra (hunk 2 above)
Comment (labradford) [reply env/.env.prod.hydra:13]: 1. You will need these files to seed the db
2. These were already in the repo in the env.dev.db so they aren't true production values
we can remove these before committing this MR, but they are here to get the app running.

**Review by labradford — COMMENTED** [review]
Note: once https://github.com/scientist-softserv/west-virginia-university/pull/48 is merged, we will need to add BULKRAX_USERNAME & BULKRAX_PW env vars to the server with values of their choice

**Pull request closed without merging**